dist/jquery.floatThead-slim.js

```diff
@@ -482,15 +514,20 @@
               cells.push('<th class="floatThead-col"/>');
             }
             cols.push('<col/>');
-            psuedo.push("<fthtd style='display:table-cell;height:0;width:auto;'/>");
+            psuedo.push(
+              $('<fthtd>').css({
+                'display': 'table-cell',
+                'height': 0,
+                'width': 'auto'
+              })
+            );
           }
 
           cols = cols.join('');
           cells = cells.join('');
 
           if(createElements){
-            psuedo = psuedo.join('');
-            $fthRow.html(psuedo);
+            $fthRow.append(psuedo);
```

Comment:
is appending an array actually any faster than `html` which i assume uses innerHTML? 

I am optimizing for speed vs clarity in most cases here.